timApp/velp/velp.py

```diff
@@ -711,7 +714,21 @@ def create_velp_group_route(doc_id: int) -> dict:
             new_group_path
         )  # Check name so no duplicates are made
         if group_exists is None:
-            original_owner = target.owners[0]
+            # Document may not have an owner, we have to account for that
+            if target.owners:
+                original_owner = target.owners[0]
+            else:
+                # TODO Should owner default to folder owner in this case? These groups will not be visible
+                #      without sufficient folder rights, however. Otherwise we could end up checking for
+                #      owners until the root of the user folder.
+                # target = Folder.find_by_path(doc_path)
+                # if not target:
+                #     raise RouteException(f"Folder not found: {doc_path}")
+                # doc_name = ""
+                # original_owner = target.block.owners[0]
```

Comment:
TODO:n voi jättää (ja on ihan hyvä, että se on), mutta kommentoitu koodi ehkä pois. Jos on tarve laittaa joku koodi "talteen", niin mielellään esim. niin, että puskee sen omaan haaraan tai tekee commitin ja peruuttaa sen. 

Pitäisin itse mielellään kommentit vain sanallisia kommentteja varten. "Work in Progress" tai "Proof of Concept" -koodit voi laittaa ihan koodina haaraan tai committiin, jos se koodipätkä on sen verran monimutkainen ettei sitä saa keksittyä suoraan kommentin tai TODOn perusteella.